### DISCUSSION TITLE
Volume Control Increment Auto-Scaling on Cinnamon

### DISCUSSION THREAD

**Discussion by demonspork** [linuxmint/discussions#396] [Ideas]
In a previous discussion, Linux Mint developers noted they were not interested in adding more "tuning" options to Cinnamon related to the volume percentage when using volume adjustment keys because in other places they are removing "tuning" options to simplify the interface. I disagree and believe that this particular setting shouldn't fall into that category, but if that is the opinion of the Mint team, try this:

Set the volume increment to 1% and auto-scale the increment  based on how frequently the key is pressed and how close to extremes. Like if the volume is close to maximum, going up should be weighted to stay at 1% increments but going down should easily scale to moving down in larger jumps. Inverse for low volume levels. I believe windows 11 uses a similar method, where if you tap it a few times in a row it increases the amount it moves at a time.

**Comment by Secret-chest** [comment]
> I fundamentally believe that this particular setting should fall into that category

But why? Even though you don't always see it like you see themes, most people use it everyday, so it has a noticeable impact. When a setting affects everyday use, I think it can't be considered just tuning, and it's not like users are forced to set it.

**Reply by demonspork** [reply]
I mistyped. I disagree with the previous discussions and the development conclusion and think it should be editable. This issue is created to 1) make my disagreement known and 2) make a proposal to improve volume control without having to create a new setting.

I appreciate the mint developers keeping cinnamon settings minimalist and navigable, but i believe they are frequently going overboard, to the point of making the interface more difficult to use

**Comment by demonspork** [comment]
Clicked wrong
